shell/scripts/build-pkg.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
+source scripts/version
```

Comment:
This script will be used outside of drone CI. Could changes to this script become part of a `drone` mode which leaves the current functionality unchanged?

Reply:
Sorry, I don't understand what this paragraph means?  Does it mean that you want me to use it directly in build-pkg.sh.
like:
```
COMMIT_BRANCH=$(git rev-parse --abbrev-ref HEAD | sed -E 's/[^a-zA-Z0-9.-]+/-/g')
COMMIT=$(git rev-parse --short HEAD)
```

Reply:
Plugin developers (across all plugins) will run `shell/scripts/build-pkg.sh` on local machines. In that environment anything drone related will cause the script to fail. 

I was wondering if there could be some kind of `drone` mode for the script which gates the changes made in this PR. So by default it runs as before, but when in drone mode it does the PR changes

Reply:
If by local machine you mean running `shell/scripts/build-pkg.sh` in an environment that does not include git information，   then I guess it will fail to run, if it contains git information (The code obtained by `git clone ***`), then it should always succeed under normal circumstances.

Reply:
The local environment won't have `scripts/version` and will also want the package versioned via `package.json` rather than `latest`